specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```diff
@@ -590,6 +590,18 @@
           "type": "integer",
           "default": 10
         },
+        "cooldownPeriod": {
+          "format": "int32",
+          "description": "Optional. KEDA Cooldown Period. Defaults to 300 if not set.",
```

Comment:
```suggestion
          "description": "Optional. KEDA Cooldown Period. Defaults to 300 seconds if not set.",
```

Reply:
Updated description per request in commit https://github.com/Azure/azure-rest-api-specs/pull/30154/commits/d3a73a221d18a474a52515f11434662db96627fa